ssas/service/admin/router.go

```diff
@@ -0,0 +1,16 @@
+package admin
+
+import (
+	"net/http"
+
+	"github.com/CMSgov/bcda-app/bcda/monitoring"
```

Comment:
This is the NewRelic monitoring, iirc. I don't think we're going to use NewRelic in this project, just Splunk.